tests/cli.rs

```diff
@@ -101,3 +103,141 @@ fn basic_options() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+fn clear_session_file(name: String) -> std::io::Result<()> {
+    // Clear session file
+    let mut config_dir = match dirs::config_dir() {
+        None => panic!("couldn't get config directory"),
+        Some(dir) => dir,
+    };
+    config_dir.push("ht");
+    config_dir.push("sessions");
+    config_dir.push("httpbin.org");
+    let hash = Blake2b::new().chain(name).finalize();
+    config_dir.push(format!("{:x}", hash).get(0..10).unwrap().to_string());
```

Comment:
Possibility for this and the actual utility function to get out of sync.
Can this test utility function just use the actual util logic?

Reply:
Well, I'd like to, but I don't know how to do that in Rust without having to create a new crate for the `utils` functions. :(

Reply:
Maybe the `config` dir could be made to point to the `temp` directory during tests? That way, any created file will be deleted regardless of whether tests have failed or not.